wolox-books/src/app/screens/auth/screens/book-list/book-list.component.ts

```diff
@@ -1,15 +1,30 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { BooksService } from 'src/app/services/books.service';
+import { Subscription } from 'rxjs';
+import { IBook } from 'src/app/models/book.interface';
 
 @Component({
   selector: 'app-book-list',
   templateUrl: './book-list.component.html',
-  styleUrls: ['./book-list.component.scss']
+  styleUrls: ['./book-list.component.scss'],
 })
-export class BookListComponent implements OnInit {
+export class BookListComponent implements OnInit, OnDestroy {
+  bookSubscription: Subscription;
+  books: Array<IBook>;
 
-  constructor() { }
+  constructor(private booksService: BooksService) {}
 
   ngOnInit(): void {
+    this.bookSubscription = this.booksService.getBooks().subscribe((response) => {
+      this.books = response.page;
+    });
   }
 
+  ngOnDestroy(): void {
+    this.bookSubscription.unsubscribe();
```

Comment:
For an http client request don't need to unsubscribe

Reply:
Done 😄